src/dmd/target.d

```diff
@@ -767,6 +768,14 @@ struct Target
 
         switch (name.toDString) with (TargetInfoKeys)
         {
+            case "allKeys":
+            {
+                Expressions* exps = new Expressions();
```

Comment:
You can probably set the dim in the ctor or, other option, reserve, based on the length of `enum keys = [__traits(allMembers, TargetInfoKeys)];`

Reply:
True but backend can (i.e. will, in the case of LDC) conditionally add keys so I'm not sure of the benefit of pre-reserving anything, this code won't ever be used for DMD (aside testing).